src/page-modules/assistant/assistant.module.css

```diff
@@ -0,0 +1,96 @@
+.wrapper {
+  background-color: var(--static-background-background_accent_0-background);
+}
+
+.container {
+  background-color: var(--static-background-background_accent_0-background);
+  height: 100%;
+  position: relative;
+  display: grid;
+  grid-template-areas:
+    'main'
+    'alternatives';
+}
+
+.main {
+  grid-area: main;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: var(--spacings-xLarge);
+  width: 100%;
+  max-width: var(--maxPageWidth);
+  padding: var(--spacings-xLarge);
+  margin: 0 auto;
+}
+
+.input {
+  display: flex;
+  flex-direction: column;
+  gap: 6px;
+}
+
+.button {
+  text-align: center;
+}
+
+.heading {
+  color: var(--static-background-background_accent_0-text);
+  margin: 10px;
```

Comment:
`rem` or the spacings variables. `10px` isn't one of the set variables IIRC, so not sure how we should handle that.